homeassistant/components/onewire/onewirehub.py

```diff
@@ -120,9 +132,23 @@ def _discover_devices_sysbus(self) -> list[OWDeviceDescription]:
         """Discover all sysbus devices."""
         devices: list[OWDeviceDescription] = []
         assert self.pi1proxy
-        for interface in self.pi1proxy.find_all_sensors():
+        all_sensors = self.pi1proxy.find_all_sensors()
+        if not all_sensors:
+            _LOGGER.error(
+                "No onewire sensor found. Check if dtoverlay=w1-gpio "
+                "is in your /boot/config.txt. "
+                "Check the mount_dir parameter if it's defined"
+            )
+        for interface in all_sensors:
             device_family = interface.mac_address[:2]
             device_id = f"{device_family}-{interface.mac_address[2:]}"
+            if device_family not in DEVICE_SUPPORT_SYSBUS:
```

Comment:
So this is where I have a question.  Won't this only fire if the entire family is unknown?  Ie, In the case prior to me adding HB_HUB to binary_sensors.py, EF is a known family, so it will pass that test, but the hub itself is an unknown device.  How will we pick that up?  Like I know he made a few sensors I don't own.  How will we tell the user "yeah, we support HobbyBoards, but not *that* HobbyBoard" ?

Reply:
In SysBus/Direct mode, then the check is done only on the device family, but in OWServer mode, then the check is done on the underlying type for EDS and HobbyBoard:
https://github.com/home-assistant/core/blob/3d79e90c9dd2f911f4673fd25a41ad980b97a9f7/homeassistant/components/onewire/onewirehub.py#L58-L62

Reply:
Since at this stage EDS and HobbyBoard devices are not supported at all in direct mode, this should be fine.

Reply:
OK, ok, I get it.  Took me a few to work through that.